src/drivers/ins/microstrain/MicroStrain.cpp

```diff
@@ -369,6 +638,12 @@ bool MicroStrain::initializeIns()
 		return false;
 	}
 
+	// Get the supported descriptors for the device in use
+	if (getSupportedDescriptors() != MIP_ACK_OK) {
```

Comment:
This and all other configuration (not having to do with baudrate) should happen after setting the baudrate to the desired value. The log after this makes me think that this is happening before that